sygnal/apnspushkin.py

```diff
@@ -311,8 +311,9 @@ async def _dispatch_notification_unlimited(
             if device.data:
                 default_payload = device.data.get("default_payload", {})
                 if not isinstance(default_payload, dict):
-                    log.error(
-                        "default_payload is malformed, this value must be a dict."
+                    logger.warning(
+                        "Rejecting pushkey due to misconfigured default_payload, "
+                        "please ensure that default_payload is a dict."
                     )
```

Comment:
I'm confused about the difference between `log` and `logger`. What's the difference between them?

Reply:
Looks like I was confused too, but after looking at this, `logger` is the original logger which is instantiated on line 70, and gets passed in to `_dispatch_notification_unlimited` where it is used to instantiate `log`, which is a `NotificationLoggerAdaptor`, on line 325. So line 314 should really still use `log` instead of `logger`, thanks for catching that.